ci-operator/step-registry/openshift/e2e/test/network-qe/openshift-e2e-test-network-qe-chain.yaml

```diff
@@ -0,0 +1,20 @@
+chain:
+  as: openshift-e2e-test-network-qe
+  env:
+  - name: TEST_SCENARIOS
+    default: SDN
+  - name: E2E_RUN_TAGS
+    default: FDP
```

Comment:
@anuragthehatter per your PR, you want to run FDP case, but here you use SDN to select case. it means you select SDN's case.

if you want to run FDP case, you need 
1, add FDP in your case title, not in case's other attribution
for example, you have case `Author:anusaxen-High-54714-Check Ingress Firewall Allow/Deny functionality for TCP via Nodeport svc`, you could add FDP as `Author:anusaxen-High-54714-[FDP]Check Ingress Firewall Allow/Deny functionality for TCP via Nodeport svc`
2, and then set FDP in TEST_SCENARIOS

and also, the default filter of openshift-extended-test is 
`~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly`, so you'd better to change the following to include Disruptive and Serial case.
`~ChkUpgrade&;~NonPreRelease&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly`

so, the setting for golang step seems
```yaml
      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly
      TEST_SCENARIOS: FDP
      TEST_TIMEOUT: "35"
```
put internal FDP as TEST_SCENARIOS

or
```yaml
      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;FDP&
      TEST_SCENARIOS: SDN
      TEST_TIMEOUT: "35"
```
put internal FDP as TEST_FILTERS

I prefer to the second.


NOTE: FDP is too short, better to make it longer, like `FDPXX`, XX is other capital char.

Reply:
Make sense. Thanks @kuiwang02

Reply:
we need serial as well so new filters look like
```
      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;FdpOvnOvs&
```